### PR TITLE
First pass in MicroK8s on s390x blog

### PR DESCRIPTION
**Pull request by ktsakalozos**

### CODE REVIEW

File: content/en/blog/_posts/microk8s-s390x.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Yes, you got it right. These are all the architectures you can build a snap for; just by ticking a checkbox! Your snap can support the usual, x86–64 and ARM but also IBM’s s390x, and PowerPC. I cannot wait to get my hands on a RISC-V build.
+
+### Weekend project!
+
+Do not waste any more time. Put your software in a snap and sail to the “Great Sea of Opportunity”.
+
+*Enjoy*
+
+
```
Comment (evilnick) [inline content/en/blog/_posts/microk8s-s390x.md:40]: You should probably put your name and title at the bottom

**Review by evilnick — CHANGES_REQUESTED** [review]
Please add name/title. I also don't seem to have edit permissions on your PR so I can't edit it.

**Comment by ktsakalozos** [comment]
> Please add name/title. I also don't seem to have edit permissions on your PR so I can't edit it.

Just added the name and affiliation on the blog. Also I think I turned on access to the maintainers.

**Review by evilnick — COMMENTED** [review]
:cake:

**Review by evilnick — APPROVED** [review]

**Pull request merged**